bigquery/main.go

```diff
@@ -83,16 +103,49 @@ func (bqf *actualBQFactory) Make(ctx context.Context) (bq, error) {
 	}
 	bqClient, err := bigquery.NewClient(ctx, projectID)
 	if err != nil {
-		return nil, fmt.Errorf("Error intiializing bigquery client: %v", err)
+		return nil, fmt.Errorf("Error intializing bigquery client: %v", err)
 	}
 	newClient := &actualBQ{client: bqClient}
 	return newClient, nil
 }
 
+// GetImageSize calculates the total compressed size of an image by summing it's individual layers
+func GetImageSize(layers []v1.Layer) (*big.Rat, error) {
+	totalSum := int64(0)
+	for _, layer := range layers {
+		layerSize, err := layer.Size()
+		if err != nil {
+			return &big.Rat{}, fmt.Errorf("Error parsing layer %v: %v", layer, err)
+		}
+		totalSum += layerSize
+	}
+	return big.NewRat(totalSum, int64(1000000)), nil
```

Comment:
I assume this is the denominator for MB? Wanna make this a constant for better readability?